src/mcts/search.cc

```diff
@@ -384,9 +403,10 @@ void Search::SendMovesStats() const REQUIRES(counters_mutex_) {
   }
 }
 
-NNCacheLock Search::GetCachedNNEval(Node* node) const {
-  if (!node) return {};
+NNCacheLock Search::GetCachedNNEval(const Node* n) const {
+  if (!n) return {};
```

Comment:
Why variable rename just to rename it back to `node` 1 line below? :)

Reply:
Oh duh. It's a pointer to a const not a const pointer.

Reply:
(I probably ran into some compiler error and "fixed" 2 things at once but only one was necessary. 😝 )